src/Microsoft.DotNet.Build.Tasks/Targets/tests.targets

```diff
@@ -30,6 +30,7 @@
     <XunitCommandLine>xunit.console.netcore.exe $(TargetFileName)</XunitCommandLine>
     <XunitOptions>$(XunitOptions) -xml $(XunitResultsFileName)</XunitOptions>
     <XunitOptions>$(XunitOptions) -notrait category=failing</XunitOptions>
+    <XunitOptions>$(XunitOptions) -notrait category=nonwindowstests</XunitOptions>
```

Comment:
Why would you set this all the time? I would expect this to be conditioned in some way.